images/hub/jupyterhub_config.py

```diff
@@ -186,6 +185,8 @@
 else:
     raise ValueError("Unhandled auth type: %r" % auth_type)
 
+c.OAuthenticator.scope = get_config('auth.scopes', [])
```

Comment:
This overrides the default of the chosen OAuthenticator to an empty list if unset in the helm chart.

To avoid this, use a condition:

```python
auth_scopes = get_config('auth.scopes')
if auth_scopes:
    c.OAuthenticator.scope = auth_scopes
```

Reply:
Fixed